### ISSUE TITLE
Accessibility attributes have been lost on the Initialization screen

### ISSUE THREAD

**Issue by nodlesh** [bcgov/bc-wallet-mobile#648]
### Steps to Reproduce
Somewhere along the way the accessibility of the initialization screen changed. It used to have an accessibility label of “In progress” and the indicator element was accessible. Now it seem it is not.

On iOS used to be this,
```
<XCUIElementTypeActivityIndicator type="XCUIElementTypeActivityIndicator" value="1" name="com.ariesbifold:id/LoadingActivityIndicator" label="In progress" enabled="true" visible="true" accessible="true" x="177" y="397" width="21" height="21" index="0"/>\n
```

Somewhere along the way it changed to this,
```
<XCUIElementTypeOther type="XCUIElementTypeOther" name="com.ariesbifold:id/LoadingActivityIndicator" enabled="true" visible="true" accessible="false" x="102" y="258" width="171" height="171" index="0">
Is this an issue or have we done this on purpose for good reason?
```

### Expected Result
Accessibility label and accessible flag should be set. 

### Environment
All devices

### Workaround
No workaround known

### Severity
Medium

### Priority
Medium

### Test Impact
None

**Comment by jleach** [comment]
@nodlesh Is that label needed? I'd only add accessibility labels to something the user needs to interact with.

**Comment by jmdkumar08** [comment]
Won't fix Label
